packages/mobile/src/components/now-playing-drawer/ActionsBar.tsx

```diff
@@ -168,7 +168,7 @@ export const ActionsBar = ({ track }: ActionsBarProps) => {
         isDarkMode={isDarkMode}
         iconLightJSON={[IconRepostOnLight, IconRepostOffLight]}
         iconDarkJSON={[IconRepostOnDark, IconRepostOffDark]}
-        isActive={track?.has_current_user_reposted ?? false}
+        iconIndex={track.has_current_user_reposted ? 1 : 0}
```

Comment:
Should we always be using `iconIndex` instead of `isActive`?

Reply:
Yes but the FavoriteButton and RepostButton components use isActive to populate iconIndex lol. Need to clean that up sometime, but they have different base colors.